Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -423,7 +423,7 @@ void FunctionDefinitionWriter::emitNecessaryHelpers()
 
     if (m_shaderModule.usesWorkgroupUniformLoad()) {
         m_body.append(m_indent, "template<typename T>\n"_s,
-            m_indent, "static T __workgroup_uniform_load(threadgroup T* const ptr)\n"_s,
+            m_indent, (metalAppleGPUFamily() >= 9 ? "[[clang::optnone]] "_s : ""_s), "static T __workgroup_uniform_load(threadgroup T* const ptr)\n"_s,
```

Comment:
This depends on `https://github.com/WebKit/WebKit/pull/45322` for `metalAppleGPUFamily` definition

Reply:
When I look at the generated code, it appears this is a shader validation bug. So once https://github.com/WebKit/WebKit/pull/45322 lands we should conditionalize this for SV-only just so we don't see these false positives.